src/dom/dom.js

```diff
@@ -3296,30 +3530,38 @@ class MediaElement extends p5.Element {
     this._modified = value;
   }
   /**
-   * Schedule an event to be called when the audio or video
-   * element reaches the end. If the element is looping,
-   * this will not be called. The element is passed in
-   * as the argument to the onended callback.
+   * Sets a function to call when the audio/video reaches the end of its
+   * playback. The function won't be called if the media is looping.
+   *
+   * The `p5.MediaElement` is passed as an argument to the callback function.
    *
    * @method  onended
    * @param  {Function} callback function to call when the
-   *                             soundfile has ended. The
-   *                             media element will be passed
-   *                             in as the argument to the
-   *                             callback.
+   *                             soundfile ends. The `p5.MediaElement`
+   *                             will be passed in as the argument.
    * @chainable
    * @example
-   * <div><code>
+   * <div>
+   * <code>
    * function setup() {
-   *   let audioEl = createAudio('assets/beat.mp3');
-   *   audioEl.showControls();
-   *   audioEl.onended(sayDone);
-   * }
+   *   background(200);
+   *
+   *   textAlign(CENTER);
+   *   text('Playing...', 50, 50);
+   *
+   *   // Create a p5.MediaElement using createAudio();
+   *   let beat = createAudio('assets/beat.mp3');
+   *   beat.autoplay(true);
```

Comment:
We might need it to be muted for this to work reliably. Maybe doing video instead of audio for a lot of these examples would let us keep the autoplay behaviour while muted?